apps/api/src/app/shared/services/bullmq/Bullmq.service.ts

```diff
@@ -0,0 +1,55 @@
+import { JobsOptions, Processor, Queue, QueueBaseOptions, QueueOptions, QueueScheduler, Worker } from 'bullmq';
+
+export class BullmqService {
+  private readonly _queue: Queue;
+  private _worker: Worker;
+  private _queueScheduler: QueueScheduler;
+  private readonly pro: boolean = process.env.NPM_TASKFORCESH_TOKEN !== undefined;
+
+  constructor(name: string, config: QueueOptions) {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    const QueueClass = !this.pro ? Queue : require('@taskforcesh/bullmq-pro').QueuePro;
```

Comment:
Not sure I follow...

Reply:
I think, but not 100% sure, the way `require` works is that how it is proposed now until runtime when we hit that line and we need to require we wouldn't notice if the dependency `@taskforcesh/bullmq-pro` has been installed of not.

Reply:
The problem with this is that even the community needs to buy the pro version if I do not understand you wrong...

Reply:
Maybe I can pick the PR and do a proposal PR so I can clarify what I mean. Will try to do later this week if possible.

Reply:
@p-fernandez @davidsoderberg it might be only a problem when the invalid token is set on the `NPM_TASKFORCESH_TOKEN` env variable... then the flag will be `true` and the package `@taskforcesh/bullmq-pro` won't be installed from the `@taskforcesh` registry (because it's defined as optional).